spam/02_spam_data_augmentation_tutorial.py

```diff
@@ -16,40 +16,45 @@
 # 4. **Training A Model**: We use the augmented training set to train an LSTM model for classifying new comments as `SPAM` or `HAM`.
 
 # %% [markdown]
-# ### Data Splits in Snorkel
-#
-# We split our data into 3 sets:
-# * **Training Set**: The largest split of the dataset. These are the examples used for training, and also the ones that transformation functions are applied on.
-# * **Validation Set**: A labeled set used to tune hyperparameters and/or perform early stopping while training the classifier.
-# * **Test Set**: A labeled set for final evaluation of our classifier. This set should only be used for final evaluation, _not_ tuning.
-
-# %% [markdown]
-# ## 1. Loading Data
-
-# %% [markdown]
-# We load the Kaggle dataset and create Pandas DataFrame objects for each of the sets described above.
-# The two main columns in the DataFrames are:
-# * **`text`**: Raw text content of the comment
-# * **`label`**: Whether the comment is `SPAM` (1) or `HAM` (0).
-#
-# For more details, check out the labeling functions [tutorial](https://github.com/snorkel-team/snorkel-tutorials/blob/master/spam/01_spam_tutorial.ipynb).
+# This next two cell takes care of some notebook-specific housekeeping.
+# You can ignore it.
 
 # %%
 import numpy as np
 import os
 import random
 
+# Make sure we're running from the spam/ directory
+if os.path.basename(os.getcwd()) == "snorkel-tutorials":
+    os.chdir("spam")
+
+# Turn off TensorFlow logging messages
+os.environ["TF_CPP_MIN_LOG_LEVEL"] = "3"
+
 # For reproducibility
-os.environ["PYTHONHASHSEED"] = "0"
+seed = 0
+os.environ["PYTHONHASHSEED"] = str(seed)
 np.random.seed(0)
 random.seed(0)
 
-# Turn off TensorFlow logging messages
-os.environ["TF_CPP_MIN_LOG_LEVEL"] = "3"
+# %% [markdown]
+# This next cell makes sure a spaCy English model is downloaded.
+# If this is your first time downloading this model, restart the kernel after executing the next cell.
 
-# Make sure we're running from the spam/ directory
-if os.path.basename(os.getcwd()) == "snorkel-tutorials":
-    os.chdir("spam")
+# %%
+# Download the spaCy english model
+# ! python -m spacy download en_core_web_sm
+
+# %% [markdown]
+# ## 1. Loading Data
+
+# %% [markdown]
+# We load the Kaggle dataset and create Pandas DataFrame objects for each of the sets described above.
```

Comment:
objects for the train, test and validation sets (There are no sets described above anymore.)